kimjaegwan/src/components/Comment/Comment.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import styleSet from '../../styles/styleSet';
+import Image from 'next/image';
+import user from '../../imgs/user.png';
+
+interface IComment {
+    id: number;
+    name: string;
+    content: string;
+}
+
+function Comment(Props: IComment) {
```

Comment:
```suggestion
function Comment(props: IComment) {
```
소문자로 적는 것이 더 관례적인 선택일 거 같아요 ~

> 사실 프로젝트에서는 eslint와 같은 도구로 컨벤션 설정을 한답니다

Reply:
인터페이스를 사용해서 하면 더 좋네요..담부터 열심히 써봐야겠어요